my_list.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*args)
+    @list = args
+  end
+end
```

Comment:
- [ ] Please remember that you are supposed to implement your own `each` method, you should not use the `each` method given by ruby, this projet is supposed to help you understand how enumerables work, kinldy change this and implement the new `each`method created by you, on the other enumerables 😄
    - You could create your own `each` method using `while` (for example).